packages/react-experiments/src/components/UnifiedPicker/UnifiedPicker.tsx

```diff
@@ -253,7 +257,7 @@ export const UnifiedPicker = <T extends {}>(props: IUnifiedPickerProps<T>): JSX.
   };
 
   const _onDragEnd = (item?: any, event?: DragEvent): void => {
-    if (event) {
+    if (event && !isInDropAction) {
       // If we have a move event, and we still have selected items (indicating that we
```

Comment:
Maybe expand the comment to include what !isInDropAction means?

Reply:
cool call, adding one